prov/efa/src/rdm/rxr_env.c

```diff
@@ -70,15 +70,18 @@ struct rxr_env rxr_env = {
 };
 
 
-/* @brief: Define FI_EFA_USE_DEVICE_RDMA as a configuration parameter
+/** @brief: Set default value of using device's RDMA
```

Comment:
Set -> Get

Reply:
I will change the name of the function to set instead of get because this function sets the default value.

Reply:
I think get is better if the function returns the value. Being pedantic... 'set' implies that the caller needs to pass in a pointer to be set.